### PR TITLE
Add FAQ to readme

### PR DESCRIPTION
**Pull request by myshov**
I stumbled upon on some issues with check-dts. So just wanted to make the troubleshooting a bit easier if other users will have the same issues.

### CODE REVIEW

**Comment by myshov** [comment]
Sure! I will the same formatting as in autoprefixer

**Comment by myshov** [comment]
@ai done :ok_hand:

File: README.md [hunk 1]
```diff
@@ -125,3 +125,19 @@ You can test only specific files by:
 $ npx check-dts **/*.tsx?
 $ npx check-dts **/*.ts !**/index.ts
 ```
+
+## FAQ
+
+### Does Autoprefixer polyfill Grid Layout for IE?
```
Comment (ai) [inline README.md:131]: You forgot to change the question :)

Comment (myshov) [reply README.md]: haha! Yes I fixed it

File: README.md [hunk 2]
```diff
@@ -125,3 +125,19 @@ You can test only specific files by:
 $ npx check-dts **/*.tsx?
 $ npx check-dts **/*.ts !**/index.ts
 ```
+
+## FAQ
+
+### It seems that check-dts ignores all d.ts-files. How can I fix this?
+
+Please make sure that your tsconfig.json doesn't include `skipLibCheck: true`.
+Becuase with this [option](https://www.typescriptlang.org/tsconfig#skipLibCheck) all d.ts-files will be ignored.
```
Comment (ai) [inline README.md:134]: Keep 80 columns line. For instance, by moving link’s URL under the text (check Autoprefixer docs).

Comment (myshov) [reply README.md]: done! :ok_hand:

File: README.md [hunk 3]
```diff
@@ -125,3 +125,20 @@ You can test only specific files by:
...
 $ npx check-dts **/*.ts !**/index.ts
 ```
+
+## FAQ
+
+### It seems that check-dts ignores all d.ts-files. How can I fix this?
+
+Please make sure that your tsconfig.json doesn't include `skipLibCheck: true`.
+Becuase with this [option](https://www.typescriptlang.org/tsconfig#skipLibCheck)
+all d.ts-files will be ignored.
```
Comment (ai) [inline README.md:135]:
```suggestion
all `d.ts`-files will be ignored.
```

File: README.md [hunk 4]
```diff
@@ -125,3 +125,20 @@ You can test only specific files by:
 $ npx check-dts **/*.tsx?
 $ npx check-dts **/*.ts !**/index.ts
 ```
+
+## FAQ
+
+### It seems that check-dts ignores all d.ts-files. How can I fix this?
+
+Please make sure that your tsconfig.json doesn't include `skipLibCheck: true`.
```
Comment (ai) [inline README.md:133]:
```suggestion
Please make sure that your tsconfig.json doesn’t include `skipLibCheck: true`.
```

File: README.md [hunk 5]
```diff
@@ -125,3 +125,20 @@ You can test only specific files by:
...
+Becuase with this [option](https://www.typescriptlang.org/tsconfig#skipLibCheck)
+all d.ts-files will be ignored.
+
+### Can I use check-dts as a simple validator for d.ts-files?
+
+Yes you can. But note if you have the typescript in your project
+you can validate whether d.ts-files have some issues with the following command:
```
Comment (ai) [inline README.md:140]:
```suggestion
you can validate whether `d.ts`-files have some issues with the following command:
```

**Comment by myshov** [comment]
@ai thanks for review! :+1:

**Comment by ai** [comment]
Thanks fir the contribution =^_^=

**Pull request merged**